apps/website/screens/components/dialog/code/DialogCodePage.tsx

```diff
@@ -35,23 +52,23 @@ const sections = [
             </td>
           </tr>
           <tr>
-            <td>onCloseClick</td>
+            <td>onBackgroundClick</td>
             <td>
               <TableCode>{"() => void"}</TableCode>
             </td>
             <td>
-              This function will be called when the user clicks the close button. The responsibility of hiding the
+              This function will be called when the on the background of the modal. The responsibility of hiding the
```

Comment:
I think this sentence has a typo: "This function will be called when the user clicks on the background of the modal."